alf/networks/critic_networks_test.py

```diff
@@ -138,6 +138,29 @@ def _train(pnet, name):
         pnet = alf.networks.network.NaiveParallelNetwork(critic_net, replicas)
         _train(pnet, "NaiveParallelNetwork")
 
+    def test_make_parallel_warning_on_naive_parallel(self):
```

Comment:
Will this be better moved to encoding_networks_test.py?

Reply:
Moved